src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -55,37 +55,43 @@
 class ContainerBuilder extends Container implements TaggedContainerInterface
 {
     /**
-     * @var ExtensionInterface[]
+     * @var array<string, ExtensionInterface>
      */
     private $extensions = [];
 
     /**
-     * @var ExtensionInterface[]
+     * @var array<string, ExtensionInterface>
      */
     private $extensionsByNs = [];
 
     /**
-     * @var Definition[]
+     * @var array<string, Definition>
      */
     private $definitions = [];
 
     /**
-     * @var Alias[]
+     * @var array<string, Alias>
      */
     private $aliasDefinitions = [];
 
     /**
-     * @var ResourceInterface[]
+     * @var array<string, ResourceInterface>
      */
     private $resources = [];
 
+    /**
+     * @var array<string, mixed>
```

Comment:
s/mixed/array (same on the method)

Reply:
I don't understand what you mean by `s/mixed/array` ?

Reply:
I see, it was a vim command. Nano user here ^^'

Reply:
vi or sed :D

Reply:
Done, I had to make this change: https://github.com/symfony/symfony/pull/41410/commits/ec090a6f75470a21e4525386d0f0a1f9db1f4f5d
To avoid the call `loadFromExtension('foo', null)`.